netbox/dcim/tables/columns.py

```diff
@@ -0,0 +1,21 @@
+import django_tables2 as tables
+
+__all__ = (
+    'SpeedColumn',
+)
+
+
+class SpeedColumn(tables.TemplateColumn):
```

Comment:
If this is the only place the column will be used, we don't need a dedicated class for it. We can just define a `TemplateColumn` on `InterfaceTable` with the appropriate template code.